docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -224,6 +224,9 @@ const buildComponentDocumentation = async (options: {
   const { filename, project, apiPagesFolder, documentedInterfaces, projects, pagesMarkdown } =
     options;
 
+  if (path.basename(filename).startsWith('use')) {
```

Comment:
Shouldn't this condition be added in `getComponentsWithApiDoc` ? To only have actual components when we reach `buildComponentDocumentation`

Reply:
Moved to the following solution in `getComponentsWithApiDoc`
```diff
       const isExported = !!project.exports[componentName];
-      if (isExported) {
+      const isHook = path.basename(file).startsWith('use');
+      if (isExported && !isHook) {
         paths.push(path.join(project.rootPath, file));
       }

```